src/test/rpc/AccountInfo_test.cpp

```diff
@@ -607,6 +632,8 @@ class AccountInfo_test : public beast::unit_test::suite
             ripple::test::jtx::supported_amendments()};
         testAccountFlags(allFeatures);
         testAccountFlags(allFeatures - featureDisallowIncoming);
+        testAccountFlags(
+            allFeatures - featureDisallowIncoming - featureClawback);
```

Comment:
why are we removing the `featureClawback` from set of all features? Shouldn't that be enabled for us to run the unit tests?

Reply:
Amendments are included by default I think. So it has already been tested on lines 633 and 634

Reply:
oh okay, got it.